build/vega-lite-schema.json

```diff
@@ -1194,7 +1194,7 @@
               "type": "null"
             }
           ],
-          "description": "Sort order for a field with discrete domain.\nThis can be `\"ascending\"`, `\"descending\"`, `null`, or a [sort field definition object](sort.html#sort-field) for sorting the field by another field.\n\n__Default value:__ `\"ascending\"`\n\n__Note:__ For fields with continuous domain, please use `\"scale\": {\"reverse\": true}` to flip the scale instead."
+          "description": "Sort order for the encoded field.\nSuppoted `sort` values include `\"ascending\"`, `\"descending\"` and `null` (no sorting).\nFor fields with discrete domains, `sort` can also be a [sort field definition object](sort.html#sort-field) for sorting the field by another field.\n\n__Default value:__ `\"ascending\"`"
```

Comment:
but you are not sorting the field. So how about sorting the domain? Or the data?